documentation/modules/deploying/con-deploy-prereqs.adoc

```diff
@@ -5,10 +5,10 @@
 [id='deploy-prereqs-{context}']
 = Deployment prerequisites
 
-To deploy Strimzi, make sure that:
+To deploy Strimzi, you'll need the following:
```

Comment:
Please refrain from using you'll. I think that is a particularly difficult one for translation.